conans/cli/commands/export.py

```diff
@@ -12,24 +12,23 @@ def common_args_export(parser):
 
 
 @conan_command(group=COMMAND_GROUPS['creator'])
-def export(conan_api, parser, *args, **kwargs):
+def export(conan_api, parser, *args):
     """
     Export recipe to the Conan package cache
     """
     common_args_export(parser)
-    add_lockfile_args(parser)
+    parser.add_argument("-l", "--lockfile", action=OnceArgument,
+                        help="Path to a lockfile.")
+    parser.add_argument("--lockfile-no-strict", action="store_true",
```

Comment:
`--lockfile-partial` ?

Reply:
It could be a complete lockfile, but you changed one ``requires = xxx`` in your recipe and use against the lockfile, want to "lock whatever you can, but if there is something you cannot lock, then evaluate it as normal". I am not sure ``-lockfile-partial`` would be closer to that than ``no-strict``

Reply:
>  "lock whatever you can, but if there is something you cannot lock, then evaluate it as normal"

So it applies the lockfile partially. Honestly, the `strict` might mean anything.

Reply:
Good for me. @czoido @franramirez688 ? ``--lockfile-partial`` better naming than ``--lockfile-no-strict``?

Reply:
I have to say I don't love `--lockfile-partial` either... maybe we need more ideas.

Reply:
I have used ``--lockfile-partial`` I agree it is slightly better than ``--lockfile-strict`` in the meaning.
If anyone else got other ideas, they are welcome.